versioned_docs/version-2.0/how_to_guides/tracing/annotate_code.mdx

```diff
@@ -26,9 +26,9 @@ If you are using LangChain (either Python or JS/TS), you can skip this section a
 LangSmith makes it easy to log traces with minimal changes to your existing code with the `@traceable` decorator in Python and `traceable` function in TypeScript.
 
 :::note
-The `LANGCHAIN_TRACING_V2` environment variable must be set to `'true'` in order for traces to be logged to LangSmith, even when using `@traceable` or `traceable`. This allows you to toggle tracing on and off without changing your code.
+The `LANGSMITH_TRACING` environment variable must be set to `'true'` in order for traces to be logged to LangSmith, even when using `@traceable` or `traceable`. This allows you to toggle tracing on and off without changing your code.
```

Comment:
Bearing in mind recent feedback about things changing without warning - can we add a callout documenting that the old variables will still work?